src/wp-includes/rest-api/endpoints/class-wp-rest-autosaves-controller.php

```diff
@@ -348,11 +348,13 @@ public function get_item_schema() {
 	 * From wp-admin/post.php.
 	 *
 	 * @since 5.0.0
+	 * @since 6.4.0 The `$meta` parameter was added.
 	 *
 	 * @param array $post_data Associative array containing the post data.
+	 * @param array $meta      Associative array containing the post meta data.
 	 * @return mixed The autosave revision ID or WP_Error.
 	 */
-	public function create_post_autosave( $post_data ) {
+	public function create_post_autosave( $post_data, $meta = array() ) {
```

Comment:
```suggestion
	public function create_post_autosave( $post_data, array $meta = array() ) {
```

We can do this, which is neat.

Reply:
Neat, does this enforce passing an array?